app/main.py

```diff
@@ -1,3 +1,75 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(self.km + other)
+        raise TypeError(
+            f"can't multiply sequence by non-int of type"
+            f" 'Distance': {other}"
+        )
+
+    def __iadd__(self, other: int | float | Distance) -> Distance:
+        if isinstance(other, (int, float)):
+            self.km += other
+            return self
+        elif isinstance(other, Distance):
+            self.km += other.km
+            return self
+        raise TypeError(
+            f"unsupported operand type(s)"
+            f" for +=: 'Distance' and {type(other).__name__}"
+        )
+
+    def __mul__(self, other: int | float) -> Distance:
+        if isinstance(other, int | float):
+            return Distance(self.km * other)
+        elif isinstance(other, (int, float)):
+            return Distance(self.km * other)
+        raise TypeError(
+            f"unsupported operand type(s)"
+            f" for *: 'Distance' and '{type(other)}'"
+        )
+
+    def __truediv__(self, other: int | float | Distance) -> Distance:
+        if isinstance(other, (int, float)):
+            return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other: Distance) -> bool:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        elif isinstance(other, (int, float)):
+            return self.km < other
```

Comment:
```suggestion
        return self.km < other
```
elif is redundant fix in all other places